front/src/containers/Authentication/styleRegister.js

```diff
@@ -0,0 +1,67 @@
+import styled from 'styled-components'
```

Comment:
I don't will look this file because i think that is the same of above. Please, do all in a unique file